doc/source/io.rst

```diff
@@ -28,6 +28,20 @@ a web URL, for example for GeoJSON files from `geojson.xyz <http://geojson.xyz/>
     url = "http://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_110m_land.geojson"
     df = geopandas.read_file(url)
 
+You can also load ZIP files that contains ESRI shapefiles::
```

Comment:
```suggestion
You can also load ZIP files that contain your files::
```

Reply:
Got it!

Reply:
I made a commit to fix this and I squashed it to the previous commit.